Overview.src.html

```diff
@@ -2760,9 +2805,15 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      <span title="handle content codings">handling content codings</span> given
      <var>codings</var> and <var>bytes</var>.
 
-     <li><p>Push <var>bytes</var> to <var>response</var>'s
-     <span title=concept-response-body>body</span>.
-     <!-- XXX xref payload body / streams -->
+     <li>
+      <p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code>
+      object wrapping an <code>ArrayBuffer</code> containing <var>bytes</var> to <var>stream</var>.
+
+      <p class="note no-backref">This operation will not throw an exception.
+
+     <li><p>If <var>stream</var> doesn't <span title=concept-ReadableStream-need-more-data>need more
+     data</span> and <var>request</var>'s <span>synchronous flag</span> is unset, ask the user agent
+     to <span title=concept-fetch-suspend>suspend</span> the ongoing fetch.
     </ol>
 
     <p class="note no-backref">This means that the `<code title>Content-Length</code>`
```

Comment:
This note now looks a bit like it's referring to step 5... I am not sure there is a good fix.

I also don't really understand the note. Why is Content-Length "no longer" reliable for responses? Was it ever reliable?